pkg/pdf/example_test.go

```diff
@@ -648,3 +648,12 @@ func ExamplePdfMaroto_SetProtection() {
 
 	// Do more things and save...
 }
+
+// ExamplePdfMaroto_SetCompression demonstrates how to disable compression
```

Comment:
Could you explain the default value of this feature?

Reply:
Comment updated

Reply:
updated readme and comments